src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -89,8 +89,6 @@ fi
 
 # ========================= BEGIN Test Execution =============================
 echo Running tests... Start time: $(date +"%T")
-echo Commands:
-[[TestRunCommandsEcho]]
```

Comment:
I removed this printing commands part as I needed to create the function `CollectDumps()` to be able to keep the exit code intact, and `echo` would have problem with parentheses.